java/core/src/java/org/apache/orc/impl/ColumnStatisticsImpl.java

```diff
@@ -1634,6 +1634,30 @@ public int hashCode() {
     }
   }
 
+  private static final class TimestampInstantStatisticsImpl extends TimestampStatisticsImpl {
+    TimestampInstantStatisticsImpl() {
+    }
+
+    TimestampInstantStatisticsImpl(OrcProto.ColumnStatistics stats) {
+      super(stats);
+    }
+
+    @Override
+    public void updateTimestamp(Timestamp value) {
+      updateTimestamp(value.getTime());
```

Comment:
do we always return the timestamp stats in local timezone? If so PPD on timestamp will still be broken right?

Reply:
I think this is correct. We will always compare the timestamp instant for the new type.